Sources/eppo/EppoClient.swift

```diff
@@ -70,20 +70,20 @@ public class EppoClient {
         }
     }
     
-    // todo: add back when supporting JSON
-//    public func getJSONAssignment(
-//        flagKey: String,
-//        subjectKey: String,
-//        subjectAttributes: SubjectAttributes,
-//        defaultValue: [String: EppoValue]) throws -> [String: EppoValue]
-//    {
-//        return try getInternalAssignment(
-//            flagKey: flagKey, 
-//            subjectKey: subjectKey, 
-//            subjectAttributes: subjectAttributes
-//        )?.variation?.value?.objectValue() ?? defaultValue
-//    }
-//    
+    public func getJSONAssignment(
```

Comment:
I think this is what we want our `getJSONStringAssignment()` to be, but will have a `getJSONAssignment()` that still returns a native JSON assignment, as Swift--and most languages--have native JSON support.

Reply:
Swift would be a [String: any] - this is a dictionary with string keys and unknown values. What else can the shape be without a struct to deserialize to? Do you think this offers a valuable object that users can parse? Easy enough to return this though.